libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -548,16 +555,18 @@ public String getNewAuthToken() {
                         }
 
                         // Broadcasts an intent that the access token has been revoked.
-                        final Intent revokeIntent = new Intent(ACCESS_TOKEN_REVOKE_INTENT);
-                        revokeIntent.setPackage(SalesforceSDKManager.getInstance().getAppContext().getPackageName());
-                        SalesforceSDKManager.getInstance().getAppContext().sendBroadcast(revokeIntent);
+                        broadcastIntent = new Intent(ACCESS_TOKEN_REVOKE_INTENT);
+                    } else if (newInstanceUrl != null && !newInstanceUrl.equalsIgnoreCase(lastNewInstanceUrl)) {
+                        // Broadcasts an intent that the instance server has changed (implicitly token refreshed too)
+                        broadcastIntent = new Intent(INSTANCE_URL_UPDATE_INTENT);
```

Comment:
I remember now why I did this. When the instance url gets refreshed, I would assume the token gets refreshed too. Is that assumption correct? If yes, then maybe I should change the name of this intent.